flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -911,46 +911,55 @@
                     <UseTemplate Name="FBW_Airbus_DCDU_Generic_Inop_Switch">
                         <NODE_ID>PUSH_MPL_MS0MINUS</NODE_ID>
                         <SEQ1_POWERED>(L:A32NX_ELEC_AC_1_BUS_IS_POWERED, Bool)</SEQ1_POWERED>
+                        <NO_SEQ2 />
```

Comment:
It would be better to just add `<NO_SEQ2 />` to the default template parameters of `FBW_Airbus_DCDU_Brightness_Switch` and `FBW_Airbus_DCDU_Generic_Switch`, instead of adding it to each usage of the template

Reply:
Done